tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -53,6 +56,13 @@ def testbed_setup(ip_version, ptfhost, rand_selected_dut, rand_unselected_dut, t
     return test_port, server_ip, ip_version
 
 
+def neighbor_reachable(duthost, neighbor_ip):
+    neigh_table = duthost.switch_arptable()['ansible_facts']['arptable']
+    ip_version = 'v4' if ip_address(neighbor_ip).version == 4 else 'v6'
+    neigh_status = neigh_table[ip_version][neighbor_ip]['state'].lower()
+    return "reachable" in neigh_status
```

Comment:
Hi @theasianpianist ,
The test failed on our setup because the state of an IPv4 neighbor is "PERMANENT" here, not "REACHABLE".
And I think that is because the neighbor is added back by "ip neighbor replace" command 
in https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/dualtor/dual_tor_utils.py#L1176
Could you tell me does this case with IPv4 pass on your setups? Do I miss something?
Thanks!

Reply:
This was fixed recently in https://github.com/sonic-net/sonic-mgmt/pull/6984, could you please retry with the changes in this PR and see if it helps?

Reply:
The fix works, thanks!